components/Form.js

```diff
@@ -77,14 +90,18 @@ export default function Form({
               </option>
             ))}
           </select>
-        </div>
-
-        <SmallText>*required</SmallText>
+        </StyledFormElement>
 
-        <Button>{prevValues?.id ? "Update" : "Create"}</Button>
-        {router.pathname === "/edit" ? null : (
-          <Button onClick={handleCancel}>Reset</Button>
-        )}
+        <StyledButtonContainer>
+          {router.pathname === "/edit" ? null : (
+            <Button type="button" onClick={handleCancel}>
```

Comment:
As this is not a canceling action but a form reset, I suggest to rename the function:

```suggestion
            <Button type="button" onClick={handleReset}>
```